daemon/mgr/container_rich_mode_test.go

```diff
@@ -0,0 +1,80 @@
+package mgr
+
+import (
+	"github.com/alibaba/pouch/apis/types"
```

Comment:
Also see 

> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

In https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md